### PR TITLE
Update make targets to fit better with what's currently in Accounting

### PR DESCRIPTION
**Pull request by kaizoku**
The `make all` target had rundev on a new line, which attempts to run it as a command, rather than making the 'all' target dependent on the 'rundev' target.

**Console Logs**:
Before fix:
```
$ make
rundev
/bin/bash: rundev: command not found
Makefile:40: recipe for target 'all' failed
make: *** [all] Error 127
```

**Merge deadline**: None

**Testing instructions**:

1. Run `make all` and ensure the `run.dev` target is run, instead of an error.

### CODE REVIEW

**Comment by UmanShahzad** [comment]
@kaizoku Yeah I saw this too, but wasn't sure if intentional or something? (That would be a hacky way to handle it. We should set the default goal to some kind of help message).

Since you'll have to rebase anyway, then when you have free time could you do this:

```Makefile
.DEFAULT_GOAL := help
HELP_SPACING ?= 30

help: ## Display this help message.
        @echo "Please use \`make <target>' where <target> is one of"
        @perl -nle'print $& if m{^[\.a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-$(HELP_SPACING)s\033[0m %s\n", $$1, $$2}'
```

and then add help messages to the commands like

```Makefile
target: ## help message
    @echo 'it works!'
```

If you don't have time let me know and I can take over the PR to do that.

**Comment by bradenmacdonald** [comment]
@kaizoku @UmanShahzad Is either of you still working on this? I like Uman's suggestion, but this also isn't something worth spending a lot of time on at the moment.

**Comment by UmanShahzad** [comment]
@bradenmacdonald I am not, at the moment. I might do it when I get free time, so if merging this particular PR isn't urgent, I can just work off of this. Whatever sounds best.

**Comment by UmanShahzad** [comment]
I'm going to take this PR over as part of https://tasks.opencraft.com/browse/OC-3615

File: Makefile [hunk 1]
```diff
@@ -17,100 +17,102 @@
...
-migration_check: clean
+migrations.check: clean ## Check for unapplied migrations
 	!(($(HONCHO_MANAGE) showmigrations | grep '\[ \]') && printf "\n\033[0;31mERROR: Pending migrations found\033[0m\n\n")
 
-migration_autogen: clean
+migrations: clean ## Generate migrations.
 	$(HONCHO_MANAGE) makemigrations
 
-run: clean migration_check collectstatic
+run: clean migrations.check static ## Run the accounting service in a production setting with concurrency.
```
Comment (UmanShahzad) [inline Makefile]: Woops, my bad.

File: Makefile [hunk 2]
```diff
@@ -17,100 +17,102 @@
...
 
-all:
-	rundev
+help: ## Display this help message.
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[\.a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-$(HELP_SPACING)s\033[0m %s\n", $$1, $$2}'
 
-apt_get_update:
-	sudo apt-get update
+all: rundev
```
[the remark this answers is not in the document]
Comment (UmanShahzad) [reply Makefile]: :+1:

File: Makefile [hunk 3]
```diff
@@ -17,100 +17,102 @@
...
-migration_check: clean
+migrations.check: clean ## Check for unapplied migrations
 	!(($(HONCHO_MANAGE) showmigrations | grep '\[ \]') && printf "\n\033[0;31mERROR: Pending migrations found\033[0m\n\n")
 
-migration_autogen: clean
+migrations: clean ## Generate migrations.
 	$(HONCHO_MANAGE) makemigrations
 
-run: clean migration_check collectstatic
+run: clean migrations.check static ## Run Ocim in a production setting with concurrency.
```
[the remark this answers is not in the document]
Comment (UmanShahzad) [reply Makefile]: :+1:

**Comment by UmanShahzad** [comment]
@itsjeyd Nice catches, thank you!

**Pull request merged**